java/greeter/docker-compose.yml

```diff
@@ -34,36 +34,21 @@ services:
   #    StateFun runtime
   ###############################################################
 
-  statefun-manager:
-    image: apache/flink-statefun:3.2.0-java11
-    expose:
-      - "6123"
+  statefun:
+    image: statefun-playground-entrypoint:latest
```

Comment:
Should probably become `ververica/statefun-playground-entrypoint:3.2.0` if adopted.

Reply:
why not `apache/statefun-playground` ?

Reply:
True, it could probably be `apache/statefun-playground`. I think I was influenced by the image for the Kafka producer that we are also hosting under `ververica/`. Should we maybe change this as well?

Reply:
Yes, good idea!